src/GramatykiGrafoweAGH/project/task2.py

```diff
@@ -0,0 +1,9 @@
+from GramatykiGrafoweAGH import Graph
+
+
+def P3(G: Graph) -> None:
```

Comment:
Why not implemented?

Reply:
IMO you should do at least P3 and P4 in this PR